1/ToDoApp/src/App.spec.js

```diff
@@ -0,0 +1,16 @@
+import { mount} from '@vue/test-utils'
+import {shallowMount} from '@vue/test-utils'
+import App from './App'
+
+describe('Component', () => {
+  test('is a Vue instance', () => {
+    const wrapper = mount(App)
+    expect(wrapper.isVueInstance()).toBeTruthy()
+  })
+  it('is rendering welcome-heading', () =>{
+    const wrapper=shallowMount(App)
+    expect(wrapper.find('h1').text()).toEqual('Simple Todo App');
+  })
+})
```

Comment:
hier könnte man den wrapper einmal am Anfang oder auch in einer beforeEach() methode erstellen 

Und was genau ist denn der Unterschied zwischen test und it? Beim kurzen Googlen sah es so aus als wären das Aliase. Wenn das wirklich so ist, wär es vielleicht cooler (styletechnisch) bzw auch konsistenter sich für eins zu entscheiden. :)

Reply:
Es sind Aliase :wink: 

Ich benutze immer `it` damit es sich besser liest: 
```js
test('is a Vue instance', () => {})
it('is a Vue instance', () => {}) // better

```